src/lib/one-graph/cli-netlify-graph.js

```diff
@@ -169,36 +191,64 @@ const generateHandler = (netlifyGraphConfig, schema, operationId, handlerOptions
     currentOperationsDoc = NetlifyGraph.defaultExampleOperationsDoc
   }
 
-  const handlerSource = NetlifyGraph.generateHandlerSource({
+  const result = NetlifyGraph.generateHandlerSource({
     handlerOptions,
     schema,
     netlifyGraphConfig,
     operationId,
     operationsDoc: currentOperationsDoc,
   })
 
-  if (!(handlerSource && handlerSource.source)) {
+  if (!result) {
+    warn(`No handler was generated for operationId ${operationId}`)
     return
   }
 
-  const { operation, source } = handlerSource
+  const { exportedFiles, operation } = result
+
+  ensureFunctionsPath(netlifyGraphConfig)
 
-  const operationName = (operation.name && operation.name.value) || 'Unnamed'
+  exportedFiles?.forEach((exportedFile) => {
```

Comment:
Sadly we need to support Node 12, and the CLI isn't compiled

Reply:
oh yea node 12 is such a burden :(

Reply:
Fixed up!